deploy/internal/deployment-endpoint.yaml

```diff
@@ -46,16 +46,22 @@ spec:
         - containerPort: 6001
         - containerPort: 6443
         env:
-        - name: JWT_SECRET
-          valueFrom:
-            secretKeyRef:
-              name: noobaa-server
-              key: jwt
         - name: MGMT_URL
+        - name: MGMT_PORT
+        - name: BG_PORT
+        - name: MD_PORT
+        - name: HOSTED_AGENTS_PORT
```

Comment:
do we need `HOSTED_AGENTS_PORT`?

Reply:
+1 , Assuming we work with n2n_proxy we probably don't right?

Reply:
These ports are moved to the other cluster to support a full router configuration, even if currently we do not invoke any API calls that are targeted in the hosted agents.
In any case, we still need to provide the server_rpc router an address (which includes a port) for the hosted_agents domain on the router

Reply:
Not using ports anymore - Changed to use full address for each service